rero_ils/modules/stats/tasks.py

```diff
@@ -49,3 +53,24 @@ def collect_stats_librarian():
             dbcommit=True, reindex=True)
         return f'New statistics of type {stat["type"]} has\
             been created with a pid of: {stat.pid}'
+
+
+@shared_task()
+def collect_stats_report():
+    """Create the montly and yearly statistics reports."""
+    today = date.today()
+    if today.day == 1:
+        with current_app.app_context():
+            stats_cfgs = StatConfiguration.get_active_stats_configurations()
```

Comment:
Is could be better to have minimum logic into `tasks` method. Could you create a `get_stats_config_to_execute(today)` method to return only `StatCfg` to execute in this task. So you could skip `if` statement into next `for` statement